kythe/rust/indexer/src/indexer/analyzers.rs

```diff
@@ -417,7 +418,8 @@ impl<'a, 'b> CrateAnalyzer<'a, 'b> {
         let defs = self.analysis.defs.clone();
 
         for def in &defs {
-            let file_vname = self.file_vnames.get(def.span.file_name.to_str().unwrap());
+            let file_name = clean(def.span.file_name.to_str().unwrap());
```

Comment:
As `def.span.file_name` should be a `PathBuf` I believe you can:

```
use path_clean::PathClean;
...
let file_name = def.span.file_name.clean();
```
directly

Reply:
There also appears to be a recent fork of `path-clean` (called `clean-path`) which may be a better choice? I'm not good at evaluating Rust library quality, though.

Reply:
I feel that `clean()` is a better use case here because it returns a String rather than another PathBuf.

Reply:
That's fine. Arguably, all of this should be using Path or PathBuf (as they are paths) rather than string, but that's a bigger change.